docker/launch-netbox.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+load_configuration() {
+  UNIT_CONFIG="/etc/unit/nginx-unit.json"
```

Comment:
Maybe we could make this variable able to be influenced from the outside?

```suggestion
  UNIT_CONFIG="${UNIT_CONFIG-/etc/unit/nginx-unit.json}"
```